src/containers/search/SearchForm.tsx

```diff
@@ -247,7 +251,7 @@ export const SearchForm: React.FC = () => {
                 size={buttonSize}
                 variant="text"
                 sx={{
-                  minWidth: '210px',
+                  minWidth: '230px',
```

Comment:
I would use mui numbers, and avoid hardcoded px (3)

Reply:
MUi numbers are great for spacing between elements. This is really styling a specific elmeent.  Therefore it also is used as `spacing(1)`. 

See: https://mui.com/material-ui/customization/spacing/